### PR TITLE
Add appveyor.yml

### PR DESCRIPTION
**Pull request by blairconrad**

### CODE REVIEW

**Comment by adamralph** [comment]
@blairconrad are you leaving the branch in place for a reason?

File: build.ps1 [hunk 1]
```diff
@@ -24,11 +24,19 @@ if ( ! ( Test-Path $NuGetExecutable ) ) {
...
+    New-Item -ItemType Directory .nuget > $nul
+}
+
 if ( ! ( Test-Path .nuget\NuGet.exe ) ) {
+    Write-Output "Copying NuGet.exe into .nuget directory"
     Copy-Item $NuGetExecutable .nuget\NuGet.exe
 }
 
 # restore packages
+Write-Output "Restoring essential NuGet packages"
```
Comment (adamralph) [inline build.ps1:39]: These packages are essential for running the build script, the other packages are essential for building. 😉 

Perhaps "Restoring NuGet packages for build script"?

**Comment by blairconrad** [comment]
> are you leaving the branch in place for a reason?

Not really. Sometimes I just  don't clean up right away. I guess this is a bigger deal when my repo is _the repo_. I've been wondering about making an organization.

File: build.ps1 (hunk 1 above)
Comment (blairconrad) [reply build.ps1:39]: Better!

**Comment by adamralph** [comment]
> I've been wondering about making an organization.

From experience, I would advise against it until you want to co-own the project with someone else. I created several organisations for my own projects before and have subsequently backtracked and moved them under my own username again.

**Comment by blairconrad** [comment]
Thanks for the advice, @adamralph. I will take you up on it!

**Pull request merged**